src/http/cookie.cr

```diff
@@ -42,9 +47,29 @@ module HTTP
       "#{@name}=#{URI.escape value}"
     end
 
+    # Returns the `Time` at which this cookie will expire, or `nil` if it will not expire.
+    # Uses *max-age* and *expires* values to calculate the time.
+    def expiration_time
+      max_age = @max_age
```

Comment:
I'd suggest to move the variable assignment into the condition.
The `elsif` branch is unnecessary because `@expires` is either `nil` or `Time` and this is what this method returns.
```
if max_age = @max_age
  creation_time + max_age
else
  @expires
end
```